providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -438,14 +490,27 @@ static int ml_kem_get_params(void *vkey, OSSL_PARAM params[])
         }
     }
 
+    p = OSSL_PARAM_locate(params, OSSL_PKEY_PARAM_ML_KEM_SEED);
+    if (p != NULL && ossl_ml_kem_have_seed(key)) {
+        if (p->data_type != OSSL_PARAM_OCTET_STRING)
```

Comment:
get_octect_string_ptr() ?

Reply:
But I'm not `getting` the value, I'm about to set it, that is write the key at the value pointer.

Reply:
But, it seems that `OSSL_PARAM_set_octet_string()` may be the right way forward, especially once https://github.com/openssl/openssl/pull/26436 is merged in a few hours...

Reply:
Actually, I can't use it, because I don't have the result buffer already in hand, I need a place to serialise the result to, and for that I need to do the checks myself.